docs/content/v0.20/example-wecs.md

```diff
@@ -8,7 +8,8 @@ register them with the hub as descibed in the
 OCM hub. This cluster will act as a workload cluster.
 
    ```shell
-   flags="--force-internal-endpoint-lookup" # set this to "" if you have installed KubeStellar on an OpenShift cluster
+   # set flags to "" if you have installed KubeStellar on an OpenShift cluster
```

Comment:
`#` is not a command in zsh.
`:` is a command in zsh, sh, and bash.

Reply:
you mean a comment, right ?

Reply:
fixed

Reply:
Technically, ':' does not mark a comment; rather, it is a command that does nothing. There are important differences. As a command, the usual command-line parsing and processing is done (rather than special skipping for comments).